crates/sui-graphql-rpc/schema.graphql

```diff
@@ -0,0 +1,823 @@
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Open Questions
+# --------------
+# - Rules for nullabilit/non-nullability
+# - Pattern for multi-gets
+# - Encoding RPC errors
+# - `type` (vs `scalar`) for Move Types
+# - Pattern for upcasting and downcasting
+
+# Check Completeness Against Existing APIs
+# -------------
+# DONE Read
+# DONE - Objects
+# DONE - Transactions
+# DONE - Checkpoints
+# DONE - Events
+# DONE - ProtocolConfig
+# DONE - Chain Identifier
+# DONE Coin
+# DONE Extended
+# DONE Governance
+# DONE Indexer
+# DONE Move Utils
+# TODO Write
+
+# DONE Port designs from Notion
+# DONE - Display
+# DONE - SuiNS
+# DONE Data Consistency API
+# TODO Flesh out Connections
+# TODO - TotalTransactionBlocks for the PTB connection
+
+schema {
+  query: Query
+  subscription: Subscription
+}
+
+type Query {
+  # First four bytes of the network's genesis checkpoint digest
+  # (uniquely defines the network)
+  chainIdentifier: String!
```

Comment:
are there other things about the network we may want to represent now or in the future?  It might be worth have a ChainMetadata type or something similar, but this is probably also okay as is